.storybook/preview.js

```diff
@@ -0,0 +1,4 @@
+
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
```

Comment:
Comments about what this does would be nice.

Reply:
Actually I don't think this is needed in this implementation.